src/i18n/en.json

```
@@ -238,7 +242,61 @@
     "BaseIncome": "Base Income",
     "Ransom": "Ransom",
-    "RansomDetails": "Details"
+    "RansomDetails": "Details",
+    "RuneMagicCard": {
+      "MissingJournalLinkTitle": "Edit the spell and add a description to have a journal link here!",
+      "CastUsingRune": "Cast using Rune:",
+      "RuneHasExperienceTitle": "Has Experience Check",
+      "RunePointCost": "Rune Point Cost:",
+      "MagicPointBoost": "Magic Point Boost:",
+      "MeditationOrRitual": "Meditation or Ritual:",
+      "MeditationOrRitualValue0": "None",
+      "MeditationOrRitualValue5": "Meditate: 1 melee round, +5% bonus",
+      "MeditationOrRitualValue10": "Meditate: 2 melee rounds, +10% bonus",
+      "MeditationOrRitualValue15": "Meditate: 5 melee rounds (1 minute), +15% bonus",
+      "MeditationOrRitualValue20": "Meditate: 25 melee rounds (5 minutes), +20% bonus",
+      "MeditationOrRitualValue25": "Meditate: 50 melee rounds (10 minutes), +25% bonus",
+      "MeditationOrRitualValue30": "Ritual: Half an hour, +30% bonus",
+      "MeditationOrRitualValue35": "Ritual: 1 hour, +35% bonus",
+      "MeditationOrRitualValue40": "Ritual: 5 hours, +40% bonus",
+      "MeditationOrRitualValue45": "Ritual: 10 hours, +45% bonus",
+      "MeditationOrRitualValue50": "Ritual: 1 day, +50% bonus",
+      "MeditationOrRitualValue55": "Ritual: 2 days, +55% bonus",
+      "MeditationOrRitualValue60": "Ritual: 1 week, +60% bonus",
+      "MeditationOrRitualValue65": "Ritual: 2 weeks, +65% bonus",
+      "MeditationOrRitualValue70": "Ritual: 4 weeks, +70% bonus",
+      "MeditationOrRitualValue75": "Ritual: 1 season, +75% bonus",
+      "MeditationOrRitualValue80": "Ritual: 1 year, +80% bonus",
+      "MeditationOrRitualValue85": "Ritual: 2 years, +85% bonus",
+      "MeditationOrRitualValue90": "Ritual: 5 years, +90% bonus",
+      "MeditationOrRitualValue95": "Ritual: 10 years, +95% bonus",
+      "MeditationOrRitualValue100": "Ritual: 20 years, +100% bonus",
+      "SkillAugmentation": "Skill Augmentation:",
+      "SkillAugmentationValue0": "None",
+      "SkillAugmentationValue50": "Critical Success, +50%",
+      "SkillAugmentationValue30": "Special Success, +30%",
+      "SkillAugmentationValue20": "Success, +20%",
+      "SkillAugmentationValue-20": "Failure, -20%",
+      "SkillAugmentationValue-50": "Fumble, -50%",
+      "OtherModifiers": "Other Modifiers:",
+      "getActorFromIdsWarning": "Couldn't get actor from actor id or token id.",
+      "noSelectedRuneWarning": "Attempt to roll a Rune Magic Spell with no Rune selected on the card.",
+      "resultMessageCritical": "The spell takes effect and costs no Rune Points and {magicPointBoost} Magic Points from boosting.",
+      "resultMessageSpecial": "The spell takes effect and costs {runePointCost} and {magicPointBoost} Magic Points from boosting.",
+      "resultMessageSuccess": "The spell takes effect and costs {runePointCost} and {magicPointBoost} Magic Points from boosting.",
```

Maybe add "rune points" to make it more clear what the cost is
```suggestion
      "resultMessageSpecial": "The spell takes effect and costs {runePointCost} Rune Points and {magicPointBoost} Magic Points from boosting.",
      "resultMessageSuccess": "The spell takes effect and costs {runePointCost} Rune Points and {magicPointBoost} Magic Points from boosting.",
```